### PR TITLE
wiki(resources): add new resources

### PR DESCRIPTION
**Pull request by Doishy**

### CODE REVIEW

File: markdown/chapters/appendices/resources.md [hunk 1]
```diff
@@ -68,8 +70,21 @@ Assume English unless specified otherwise.
 Videos from non-regular streamers or Doomsday content creators.  
 Assume English unless specified otherwise.
 
+### 2022
+
+ - Awaiting Content :D
+
```
Comment (angrybacon) [inline markdown/chapters/appendices/resources.md:76]: Probably should wait for content to add the heading imo

File: markdown/chapters/appendices/resources.md [hunk 2]
```diff
@@ -68,8 +70,21 @@ Assume English unless specified otherwise.
...
 ### 2021
 
+- [MTGBlackSet display some Paper Meandeck in action vs Death and Taxes (December 2021) - German/Deutsch](https://www.youtube.com/watch?v=Hn3no4hz00s)
+- [BoshNRoll runs a league with UB Meandeck ft. Death's Shadow (December 2021)](https://www.youtube.com/watch?v=sbx6_WIuSR4)
+- [MTGBlackSet display some Paper Meandeck in action vs Moon Stompy (November 2021) - German/Deutsch](https://www.youtube.com/watch?v=898Kqa1wiGw)
```
Comment (angrybacon) [inline markdown/chapters/appendices/resources.md:81]: Could go next to the other link (add the `s` to `display`). Minor nitpick, if "ft." takes a dot, should "vs" as well. I've always written it with personally

File: markdown/chapters/appendices/resources.md [hunk 3]
```diff
@@ -68,8 +70,21 @@ Assume English unless specified otherwise.
...
+- [Phil G of Thraben University gives their second (much improved) try at Meandeck (October 2021)](https://www.youtube.com/watch?v=5sYA9LsrVLc)
+- [Phil G of Thraben University gives their first try at Meandeck (October 2021)](https://www.youtube.com/watch?v=P-dBBwj0pFk&list=WL&index=7)
+- [NathanLipetz gives Esper Meandeck a go with kl0gW (October 2021)](https://www.youtube.com/watch?v=c5wEEhXjrS0)
+- [BoshNRoll runs a league with UB Meandeck (October 2021)](https://www.youtube.com/watch?v=t6NK3NoHg7M)
+- [Some beginner's have a go at running Meandeck for the first time (October 2021)](https://www.youtube.com/watch?v=Wz9aWgCvByA)
```
Comment (angrybacon) [inline markdown/chapters/appendices/resources.md:86]: Same thing, do you think these "beginner" leagues are good enough for the resource page?

**Review by angrybacon — CHANGES_REQUESTED** [review]

File: markdown/chapters/appendices/resources.md (hunk 1 above)
Comment (Doishy) [reply markdown/chapters/appendices/resources.md:76]: Removed

File: markdown/chapters/appendices/resources.md (hunk 2 above)
Comment (Doishy) [reply markdown/chapters/appendices/resources.md:81]: Moved and added .

File: markdown/chapters/appendices/resources.md [hunk 4]
```diff
@@ -68,8 +70,21 @@ Assume English unless specified otherwise.
...
+
+ - Awaiting Content :D
+
 ### 2021
 
+- [MTGBlackSet display some Paper Meandeck in action vs Death and Taxes (December 2021) - German/Deutsch](https://www.youtube.com/watch?v=Hn3no4hz00s)
+- [BoshNRoll runs a league with UB Meandeck ft. Death's Shadow (December 2021)](https://www.youtube.com/watch?v=sbx6_WIuSR4)
+- [MTGBlackSet display some Paper Meandeck in action vs Moon Stompy (November 2021) - German/Deutsch](https://www.youtube.com/watch?v=898Kqa1wiGw)
+- [Phil G of Thraben University gives their second (much improved) try at Meandeck (October 2021)](https://www.youtube.com/watch?v=5sYA9LsrVLc)
+- [Phil G of Thraben University gives their first try at Meandeck (October 2021)](https://www.youtube.com/watch?v=P-dBBwj0pFk&list=WL&index=7)
```
[the remark this answers is not in the document]
Comment (Doishy) [reply markdown/chapters/appendices/resources.md:83]: As per convo, leave in to show improvement!

File: markdown/chapters/appendices/resources.md (hunk 3 above)
Comment (Doishy) [reply markdown/chapters/appendices/resources.md:86]: Removed Line 86. Will leave the rest in because famous streamers.

**Review by angrybacon — APPROVED** [review]

**Review by angrybacon — APPROVED** [review]

**Pull request merged**